x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/asset_details_flyout.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React, { Dispatch, SetStateAction } from 'react';
+import AssetDetails from '../../../../../components/asset_details/asset_details';
+import { useSourceContext } from '../../../../../containers/metrics_source';
+import { useWaffleTimeContext } from '../../hooks/use_waffle_time';
+import { orderedFlyoutTabs } from './flyout_tabs';
+
+interface Props {
+  assetName: string;
+  closeFlyout: () => void;
+  setIsAlertFlyoutVisible: Dispatch<SetStateAction<boolean>>;
+}
+
+const ONE_HOUR = 60 * 60 * 1000;
+
+export const AssetDetailsFlyout = ({ assetName, closeFlyout, setIsAlertFlyoutVisible }: Props) => {
+  const { source } = useSourceContext();
+  const { currentTime } = useWaffleTimeContext();
+
+  const currentDateRange = {
+    from: new Date(currentTime - ONE_HOUR).toISOString(),
+    to: new Date(currentTime).toISOString(),
+  };
+
+  return source ? (
+    <AssetDetails
+      asset={{ id: assetName, name: assetName }}
+      assetType="host"
+      overrides={{
+        metadata: {
+          showActionsColumn: false,
+        },
+        alertRule: {
+          onCreateRuleClick: () => setIsAlertFlyoutVisible(true),
+          inventoryRuleLabel: i18n.translate('xpack.infra.infra.nodeDetails.createAlertLink', {
```

Comment:
It is customizable because we use the same component for the alert section (where it was first added): 
<img width="960" alt="image" src="https://github.com/elastic/kibana/assets/14139027/039545af-ac66-456f-b18e-e552573c906a">
iirc we changed the name to fit the host view and to have it generic if it's used outside of inventory contex (probably also saving space)

Reply:
Both open the alert flyout for the Inventory Threshold rule, this one at the top might not be needed anymore. 

I understand that this is what we have in the old flyout, but I think it will be enough if we just have the `create rule` button in the `Alerts` section. 

@roshan-elastic @jennypavlova , wdyt?

Reply:
I tried to keep it similar to what we had so if the user is used to clicking there it will be still in the same place. But you are right that maybe it is not needed anymore as we have it there. Let's open a follow-up issue for this one if we decide to change it so it's clear why it's not there (you can directly assign it to me)

Reply:
Now while testing I realized that it still can be useful to keep it - let's say you have the flyout bookmarked on the logs tab and you copy a value from the host's logs and want to create a rule for the host based on that 🤔 Not sure how valid it is to have it as host-specific as we have a menu to add the inventory rule but could be something to think about.

Reply:
> Both open the alert flyout for the Inventory Threshold rule, this one at the top might not be needed anymore.
> 
> I understand that this is what we have in the old flyout, but I think it will be enough if we just have the `create rule` button in the `Alerts` section.
> 
> @roshan-elastic @jennypavlova , wdyt?

Ah yeah, nice catch both. 

Yeah, that seems redundant at the top for the moment so let's remove.

In the more medium-term, it looks like @katrin-freihofner and @boriskirov are working on some UI patterns for actions against single entities like this:

![image](https://github.com/elastic/kibana/assets/117740680/8230e3af-f21d-479a-96d6-18ef032a4f97)

I don't think we could make that change right now but it's probably worth bearing in mind that we will probably want to adopt any consistent UI patterns that O11y uses in general...but don't bloat this issue with thinking about that right now

Reply:
Thank you both! I will remove the `Create inventory rule` link at the top. The Action drop-down menu looks nice, we can take a look in the future and adapt new UI improvements there 👍